tests/CMakeLists.txt

```diff
@@ -158,6 +160,9 @@ add_aktualizr_test(NAME uptane_key SOURCES uptane_key_test.cc PROJECT_WORKING_DI
 
 add_aktualizr_test(NAME uptane_network SOURCES uptane_network_test.cc PROJECT_WORKING_DIRECTORY)
 
+add_aktualizr_test(NAME discovery_secondary SOURCES ipsecondary_discovery_test.cc PROJECT_WORKING_DIRECTORY)
+
+
```

Comment:
+1 for a test, but it fails on my machine :(

~~~
[ RUN      ] ipsecondary_discovery.test_discovery
starting up on 127.0.0.1 port 60443

waiting to receive message
/home/phil/pro/client/aktualizr/tests/ipsecondary_discovery_test.cc:25: Failure
      Expected: secondaries.size()
      Which is: 0
To be equal to: 2
~~~